src/components/index.d.ts

```diff
@@ -315,36 +315,50 @@ export declare namespace RadioButtonGroup {
     required?: boolean
   }
 }
-
 export declare const Grid: {
   (props: any): JSX.Element
   propTypes: {
     children: React.ReactNode
-    className?: any
+    className?: string
     rowRefs: any
     columnRefs: any
   }
   defaultProps: {
-    className: any
+    className: string
   }
   displayName: string
 }
-export declare const Column: any
-export declare const Row: any
 export declare const Pagination: {
   ({
     fetchPageCallback,
     renderCallback,
   }: {
-    fetchPageCallback: any
-    renderCallback: any
+    /**
+     * Ultimately, this returns the JSON.parse'd data
+     */
+    fetchPageCallback: (pageNumber: number | string) => any
```

Comment:
should any -> object?

Reply:
Hrm, yeah, possibly huh. Did some googling and this was interesting: https://stackoverflow.com/a/58143592

I'll try adding this into https://github.com/getethos/ethos-design-system/pull/219 which is being consumed by Nora in monorepo and still a live PR :)

Reply:
Yah, it seemed to work fine! I added it into [this commit](https://github.com/getethos/ethos-design-system/pull/219/commits/a6a174529d8054a32a40d319804763c882963112) and here's the Nora example:

![image](https://user-images.githubusercontent.com/142403/71757202-e7d15480-2e48-11ea-8ea3-556dac50ff2a.png)